owa/app/utilities/localeHelper.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import _ from 'lodash';
+
+import UrlHelper from 'utilities/urlHelper';
+
+require('babel-polyfill');
+const urlHelper = new UrlHelper();
+export default class LocaleHelper {
+    fetchLocales = async () => {
+        const response = await axios.get(urlHelper.apiBaseUrl() + '/session');
+        return {
+            localeCode: response.data.locale ? this.getLocalCode(response.data.locale) : 'en',
+            allowedLocales: response.data.allowedLocales ? response.data.allowedLocales : ['en']
+        };
+    };
+
+    getLocalCode = (locale) => {
+        return locale.toLowerCase().split(/[_-]+/)[0];
+    };
+
+    getNativeNameByLocaleCode = (locale) => {
+        const localeCode = this.getLocalCode(locale);
+        const localLanguages = [
+            {code: 'en', nativeName: 'English'},
+            {code: 'es', nativeName: 'Español'},
+            {code: 'fr', nativeName: 'Français'},
+            {code: 'it', nativeName: 'Italiano'},
+            {code: 'pt', nativeName: 'Português'}
+        ];
```

Comment:
@sanishmaharjan why do you need those native names? And could you describe what this array is used for?

Reply:
Locale API only return locale code. We need to get nativeName form somewhere. 
This is similar to https://github.com/Bahmni/default-config/blob/f38cb22cb6b2a179631a290ee89c97a885b16f26/openmrs/apps/home/locale_languages.json#L1-L5

Reply:
But, Locale code return from openmrs core API `/openmrs/ws/rest/v1/session`.

Reply:
How about using https://www.npmjs.com/package/locale-code javascript library.

Reply:
@mks-d implement iso-639-1 to get language native name.